common/log_parser/os_tests/json_to_html.py

```diff
@@ -463,9 +464,11 @@ def main():
                                 # treat any other status as failure
                                 test_status = 'FAILED'
                                 break
+                            else:
+                                saw_pass = True
```

Comment:
Check exactly that there is atleast one pass in subtest and name variaable as has_passed